app/workers/figures_fetcher_worker.rb

```diff
@@ -26,12 +26,11 @@ def perform(figures_response, upload_id)
       figure.save!
     end
 
-    # callback to figures host for /cleanup
-    puts ">>>>>> figures API response"
-    puts figures_response
-    puts "<<<<<<"
+    # calls figures to delete assets
     FiguresExtractService.cleanup(@upload.id)
-    FiguresInlinerService.call(@upload.post.id)
 
+    # TODO: move this step into a batch
+    # run step to inject figures into post body
+    FiguresInlinerService.call(@upload.post.id)
```

Comment:
On first read, I thought ```Inliner``` in ```FiguresInlinerService``` was a typo. I was expecting ```FiguresInlineService```. ```Inliner``` can be hard to remember and easy to mistype. I am n=1.